spotify_player/src/cli/handlers.rs

```diff
@@ -178,8 +179,94 @@ pub fn handle_cli_subcommand(
             std::process::exit(1);
         }
         Response::Ok(data) => {
-            println!("{}", String::from_utf8_lossy(&data));
+            println!("{}", String::from_utf8_lossy(&data).replace("\\n", "\n"));
             std::process::exit(0);
         }
     }
 }
+
+fn handle_playlist_subcommand(args: &ArgMatches, socket: &UdpSocket) -> Result<()> {
+    let (cmd, args) = args.subcommand().expect("playlist subcommand is required");
+    let command = match cmd {
+        "new" => {
+            let empty = String::new();
+
+            let name = args
+                .get_one::<String>("name")
+                .expect("name arg is required");
+
+            let description = args.get_one::<String>("description").or(Some(&empty));
```

Comment:
instead of making `description` `Option<&String>` then unwrapping it later, can initialize it as a `String` instead:

```rust
let description = args.get_one::<String>("description").map(|s| s.to_owned()).unwrap_or_default();
```